eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -126,23 +126,23 @@ def write_output(self, dir_out):
         write_string_list_to_file(self.nsv_list, dir_out + '/' + config.NSV_LIST_FILE)
 
         # Contains traits without a mapping in Gary's xls
-        with utilities.open_file(dir_out + '/' + config.UNMAPPED_TRAITS_FILE_NAME, 'w') as fdw:
+        with utilities.open_file(dir_out + '/' + config.UNMAPPED_TRAITS_FILE_NAME, 'wt') as fdw:
```

Comment:
Not an error, but there's no difference between 'r' / 'rt' and 'w' / 'wt' because text mode is the default: https://docs.python.org/3/library/functions.html#open